app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -308,7 +309,14 @@ public void onClick(DialogInterface dialog, int which) {
     @Override
     protected void onResume() {
         super.onResume();
+
         settings = GoIVSettings.getInstance(MainActivity.this);
+
+        if (isPokeFlyRunning()) {
+            launchButton.setText(R.string.main_stop);
+        } else {
+            launchButton.setText(R.string.main_start);
+        }
```

Comment:
@DJCrapsody's testing suggests calling `launchButton.setText` here is not enough, it should also be called when starting/stopping the service.